sinonjs/sinon
# Upgrade Mochify to v6.5 and fix web worker issue

#### Purpose (TL;DR) - mandatory

This upgrades Mochify to the latest version and works around an issue in
Puppeteer where the web worker request isn't handled properly if request
interception is turned on.


See discussion and failing CI tests in https://github.com/sinonjs/sinon/pull/2109.

 #### Solution  - optional

A new flag was introduced in Mochify to disable request interception.

 #### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm test`
4. Observe that all test and specifically the webworker integration test are passing.

 #### Checklist for author
